code/modules/client/preference_setup/preferences_sql.dm

```diff
@@ -8,7 +12,10 @@
 	var/static/list/query_cache
 
 	// We aren't loading this category. Bye.
-	if (role_type && sql_role != role_type)
+	if (role_type && !(sql_role & role_type))
+#ifdef SQL_PREF_DEBUG
```

Comment:
These will horribly spam anyone who attempts to use them. But the information is necessary.

Also keeping them because fucking having to retype them every time this shit breaks while in development and probably while in master.